api/server.js

```diff
@@ -0,0 +1,4 @@
+const app = require("./app");
+app.listen(8080, () => {
+  console.log("Server has started!");
+});
```

Comment:
no newline found at the end of file error

Reply:
Should be okay. Unsure why it is doing that on each file. I will re add and commit